.gitignore

```diff
@@ -191,6 +191,16 @@ repos-download/
 /src/test/cypress/videos/
 /src/test/cypress/build
 
+#################################
+# Angular Build Environment Files
+#################################
+/src/main/webapp/app/environments/environment.*.ts
+
+##########################
+# Angular Build i18n Files
```

Comment:
I don't think we should name them "Build files", should we? They are part of the normal production code.